autoload/airline/extensions/fern.vim

```diff
@@ -3,7 +3,7 @@
 " vim: et ts=2 sts=2 sw=2
 
 scriptencoding utf-8
-if !get(g:, 'fern_loaded', 0)
+if !(get(g:, 'fern_loaded', 0) || get(g:, 'loaded_fern', 0))
```

Comment:
I think  `!` is missing.
I think this is correct.

```vim
if !(get(g:, 'fern_loaded', 0) || !get(g:, 'loaded_fern', 0))
```

Reply:
I would like to make it work if either `fern_loaded` or `loaded_fern` can be read, so that suggestion doesn't seem to fit.

Reply:
I suggested this code for backwards compatible compatibility reasons.
If you only want to use `fern_loaded`, then you don't need the `get(g:, 'loaded_fern', 0)` in the above code.

```vim
if get(g:, 'fern_loaded', 0) && get(g:, 'airline#extensions#fern#enabled', 1)
```

In fact, for backward compatibility, the fern.vim provides a `g:fern_load`.
What do you think?

Reply:
In summary, I don't think this renaming is necessary because of the backwards compatibility considerations on the fern side.

Reply:
Sorry, I totally thought `g:fern_loaded` was *temporarily* left for backward compatibility and will be removed in the future.

I agree @kazukazuinaina , please close.